docs/Developers/Newcomers_Guide.rst

```diff
@@ -310,7 +310,7 @@ and send it. You have successfully edited your last commit!
 
 Where ``myfork`` is explained
 `here <http://api.coala.io/en/latest/Developers/Newcomers_Guide.html#step-4-creating-a-fork-and-testing-your-changes>`_.
-The **Pull Request** will automatically update with the newest changes. 
+The **Pull Request** will automatically update with the newest changes.
```

Comment:
you'll have to squash this into the previous commit, otherwise gitmate will still not like it